bot/update_tracker.py

```
@@ -1,7 +1,7 @@
 # bot/update_tracker.py
 import os
 import json
-from datetime import datetime, timedelta
+from datetime import datetime, timedelta, time
```

Ruff

`datetime.time` imported but unused

Remove unused import: `datetime.time`

(F401)